salt/states/junos.py

```diff
@@ -158,15 +152,14 @@ def commit(name, **kwargs):
 
 
 @resultdecorator
-def rollback(name, id, **kwargs):
+def rollback(name, **kwargs):
```

Comment:
Changing an interface for a function which has been published without a deprecation cycle having been done is not allowed

Reply:
same as above

Reply:
see reply for salt/states/junos.py about deprecation cycle and changing interfaces

Reply:
@dmurphy18 I have reverted to old code itself.